src/main/resources/application.properties

```diff
@@ -4,5 +4,5 @@ spring.datasource.password=root1234
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
-jwt.expiration=300000
+jwt.expiration=3000000
```

Comment:
Do we need this changes?

Reply:
when testing in postman - I was missing the token lifetime

Reply:
Ok, in such case please don't forget to return previous value after testing